src/test/java/com/connexta/ingest/controllers/IngestControllerTests.java

```diff
@@ -110,7 +120,49 @@ public void testMissingCorrelationID() throws Exception {
   @Test
   public void testMissingAcceptVersionHeader() throws Exception {
     mockMvc
-        .perform(multipart("/ingest").file(FILE).file(METACARD))
+        .perform(
+            multipart("/ingest")
+                .file(FILE)
+                .file(METACARD)
+                .header("Last-Modified", LAST_MODIFIED_DATE))
+        .andExpect(status().isBadRequest());
+
+    verifyNoInteractions(mockIngestService);
+  }
+
+  @Test
+  public void testMissingLastModifiedHeader() throws Exception {
+    mockMvc
+        .perform(
+            multipart("/ingest").file(FILE).file(METACARD).header("Accept-Version", ACCEPT_VERSION))
+        .andExpect(status().isBadRequest());
+
+    verifyNoInteractions(mockIngestService);
+  }
+
+  @ParameterizedTest
+  @ValueSource(
+      strings = {
+        "2011-12-03", // ISO_LOCAL_DATE
+        "2011-12-03+01:00", // ISO_OFFSET_DATE
+        "10:15:30", // ISO_LOCAL_TIME
+        "10:15:30+01:00", // ISO_OFFSET_TIME
+        "2011-12-03T10:15:30", // ISO_LOCAL_DATE_TIME
+        "2011-12-03T10:15:30+01:00[Europe/Paris]", // ISO_ZONED_DATE_TIME
+        "2012-337", // ISO_ORDINAL_DATE
+        "2012-W48-6", // ISO_WEEK_DATE
+        "20111203", // BASIC_ISO_DATE
+        "Sun, 11 Jun 2017 14:32:28 GMT", // RFC_1123_DATE_TIME
+        "2017-06-11T14:32:28.120+0000", // ISO_DATE_TIME missing offset :
+      })
+  public void testInvalidLastModifiedHeader(String badDate) throws Exception {
+    mockMvc
+        .perform(
+            multipart("/ingest")
+                .file(FILE)
+                .file(METACARD)
+                .header("Accept-Version", ACCEPT_VERSION)
+                .header(LAST_MODIFIED, badDate))
         .andExpect(status().isBadRequest());
```

Comment:
❗️These tests succeed because correlationId is missing, not because the format of the `Last-Modified` header is invalid.

Reply:
😢

Reply:
Added CorrelationID